dnssec.go

```diff
@@ -370,9 +374,12 @@ func (rr *RRSIG) Verify(k *DNSKEY, rrset []RR) error {
 	if rr.Algorithm != k.Algorithm {
 		return ErrKey
 	}
-	if !strings.EqualFold(rr.SignerName, k.Hdr.Name) {
+
+	signerName := CanonicalName(rr.SignerName)
+	if !strings.EqualFold(signerName, k.Hdr.Name) {
```

Comment:
i dont think equalFold is correct for use in DNS....

Reply:
Hi，thanks for reviewing! This function use is identical to what's being used in master now: https://github.com/miekg/dns/blob/b77d1ed8e9282cadf21c4124f53a660fed55c8ca/dnssec.go#L373

What's the correct version you would suggest? Would `==` do? I can make the patch.

Reply:
@developStorm 
 [`equal`](https://github.com/miekg/dns/blob/b77d1ed8e9282cadf21c4124f53a660fed55c8ca/labels.go#L190) should be what you want.

Reply:
@monoidic Thanks! 7b480dc01dc7756cfa85848bf8290b47ebc6bba0 replaces all occurrences of EqualFold with equal.